test_framework/scripts/longhorn-setup.sh

```diff
@@ -28,9 +30,13 @@ export KUBECONFIG="${TF_VAR_tf_workspace}/templates/kube_config_3-nodes-k8s.yml"
 
 kubectl apply -f longhorn.yaml
 
+RETRIES=0
 while [[ -n "`kubectl get pods -n longhorn-system  | grep "instance-manager-.*\|longhorn-\(manager\|driver\|csi\)\|engine-image-.*" | awk '{print $3}' | grep -v Running`"  ]]; do
-  echo "Longhorn is being inatalled ... rechecking in 1m"
+  echo "Longhorn is being installed ... rechecking in 1m"
   sleep 1m
+  RETRIES=$((RETRIES+1))
+
+  if [[ ${RETRIES} -eq ${RETRY_COUNTS} ]]; then echo "Error: longhorn installation timeout"; exit 1t ; fi
```

Comment:
What's `exit 1t`?

Also, separate the typo fix into another commit.

Reply:
if the script exits with exit status not equal to zero, Jenkins will consider the step as failed, and cleanup step will be triggered.

Reply:
sorry, it was a typo